app/renderer/components/tap_target.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+module.exports = {
+    props: ['title', 'description'],
+    template: `
+    <div class="tap-target" data-activates="tap-edit">
```

Comment:
I understand `data-activates="tap-edit"``references to the tap-edit button outside of the component, shouldn't this be a prop as well so we have a decoupled component?